.travis.yml

```diff
@@ -1,9 +1,8 @@
 ---
 language: node_js
 node_js:
-  - "8"
+  - "10"
```

Comment:
I'm guessing the Node version came from upgrading to 3.16. Would it be okay to use Node 12? (I haven't used Travis CI before, but setting Node 12 on GitHub Actions worked fine on a few apps that I have.)